docs/getting-started/local_instance.md

```diff
@@ -29,9 +29,12 @@ python butler.py run_server --bootstrap
 python butler.py run_server
 ```
 
-This may take a few seconds to start. Once you see output line like
-`INFO <timestamp> admin_server.py:<num>] Starting admin server`, you should be able to navigate to
-[http://localhost:9000](http://localhost:9000) to view the web interface.
+This may take a few seconds to start. Once you see an output line like
+`INFO <timestamp> admin_server.py:<num>] Starting admin server`, you should be
+able to navigate to [http://localhost:9000](http://localhost:9000) to view the
+web interface. Note that the local instance may use ports other than 9000, such
```

Comment:
Can you hyperlink "other than 9000" to https://github.com/google/clusterfuzz/blob/master/src/local/butler/constants.py ? that has list of ports.

Reply:
Done.